rollbar/__init__.py

```diff
@@ -86,17 +86,68 @@
 except ImportError:
     AppEngineFetch = None
 
+def passthrough_decorator(func):
+    def wrap(*args, **kwargs):
+        return func(*args, **kwargs)
+    return wrap
+
 try:
     from tornado.gen import coroutine as tornado_coroutine
     from tornado.httpclient import AsyncHTTPClient as TornadoAsyncHTTPClient
 except ImportError:
-    def tornado_coroutine(func):
-        def wrap(*args, **kwargs):
-            return func(*args, **kwargs)
-        return wrap
-
+    tornado_coroutine = passthrough_decorator
     TornadoAsyncHTTPClient = None
 
+try:
+    from twisted.internet import reactor
+    from twisted.web.client import Agent as TwistedHTTPClient
+    from twisted.internet.defer import inlineCallbacks, Deferred, returnValue
+    from twisted.internet.protocol import Protocol
+    from twisted.web.http_headers import Headers as TwistedHeaders
+    from zope.interface import implements
+
+    from twisted.internet.defer import succeed
+    from twisted.web.iweb import IBodyProducer
+
+
+    class StringProducer(object):
+        implements(IBodyProducer)
+
+        def __init__(self, body):
+            self.body = body
+            self.length = len(body)
+
+        def startProducing(self, consumer):
```

Comment:
Style: Use underscores_for_names

(I realize that this is not the Twisted style but I'd like to keep the pyrollbar code consistent)